web/portal/platform/src/components/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,256 @@
+            </h3>
+            <p>
+              {_(
+                'Ivoz Provider is an Open Source solution by Irontec. In this portal you can add brands, brand operators, portals URL and much more.'
+              )}
+            </p>
+            <a href='/doc/en/administration_portal/platform/index.html '>
+              <LightButton>{_('Get started')}</LightButton>
+            </a>
+          </div>
+
+          <img src='assets/img/dashboard-welcome.svg' />
+        </div>
+      </div>
+      <div className='card activity'>
+        <div className='title'>{_('Recent activity')}</div>
+
+        <div className='content'>
+          {data.recentActivity.map((row, key) => {
+            return (
+              <div className='row' key={key}>
+                <div className='time'>{row.id}</div>
+                <div className='value'>{row.name}</div>
+              </div>
+            );
+          })}
+        </div>
+      </div>
+      <div className='card amount'>
+        <div className='img-container'>
+          <IconPlatformsDashboard />
+        </div>
+
+        <div className='number'>{data.brandNumber}</div>
+
+        <div className='name'>{_('Brand', { count: 2 })}</div>
+
+        <a href='/platform/brands' className='link'>
+          Go to brands
+        </a>
+      </div>
+
+      <div className='card amount'>
+        <div className='img-container'>
+          <IconClientsDashboard />
+        </div>
+
+        <div className='number'>{data.clientNumber}</div>
+
+        <div className='name'>{_('Client', { count: 2 })}</div>
+      </div>
+
+      <div className='card amount'>
+        <div className='img-container'>
+          <IconUsersDashboard />
+        </div>
+
+        <div className='number'>{data.userNumber}</div>
+
+        <div className='name'>{_('User', { count: 2 })}</div>
+      </div>
+
+      <div className='card licenses'>
+        <div className='title'>{_('Active call', { count: 2 })}</div>
+
+        <div className='radial'>
+          <div
+            className='circle'
+            style={
+              {
+                '--inbound': inbound,
+                '--outbound': outbound,
+              } as React.CSSProperties
+            }
+          ></div>
+          <div className='data'>
+            <div className='total'>{_('Total')}</div>
+            <div className='number'>{activeCalls?.total}</div>
+          </div>
+        </div>
+
+        <div className='legend'>
+          <div className='label'>
+            <div className='color orange'></div>
+            <div className='text'>{_('Outbound')}</div>
+          </div>
+
+          <div className='label'>
+            <div className='color red'></div>
+            <div className='text'>{_('Inbound')}</div>
+          </div>
+        </div>
+      </div>
+      <div className='card last'>
+        <div className='header'>
+          <div className='title'>Last added clients</div>
```

Comment:
This label is not translated. I could be _Last added brands_ because it's what table displays.